src/web/rustdoc.rs

```diff
@@ -598,7 +592,7 @@ mod test {
     ) -> Result<Option<String>, failure::Error> {
         assert_success(path, web)?;
         let data = web.get(path).send()?.text()?;
-        println!("{}", data);
+        println!("fetched path {} and got content {}", path, data);
```

Comment:
Wouldn't a log be better here?
```suggestion
        log::info!("fetched path {} and got content {}", path, data);
```

Reply:
This is in a test, so it doesn't make a big deal either way. But I can change it to `log` so it has the line numbers.